jira/src/test/java/org/arquillian/extension/governor/jira/ITJiraGovernorTest.java

```diff
@@ -25,6 +25,7 @@
 
 @RunWith(Arquillian.class)
 @RunAsClient
+@Jira("ARQ-831")

```

Comment:
I am not sure if I understand you. Mentioned test class contains both types of `@Jira` annotation. As class level as method level annotation.